tests/integration/cli_mint_proposal_test.go

```diff
@@ -73,15 +80,8 @@ var _ = Describe("Integration - Mint coins to given address", func() {
 
 	It("should check the proposal status to ensure it has passed", func() {
 		By("sending a QueryProposal query from `validator0` container")
-		proposal, err := cli.QueryProposal(cli.Validator0, "1")
+		proposal, err := cli.QueryProposal(cli.Validator0, Proposal_id)
 		Expect(err).To(BeNil())
 		Expect(proposal.Status).To(BeEquivalentTo(govtypesv1.StatusPassed))
 	})
-
-	It("should have the correct balance after minting", func() {
-		By("querying the balance of the given address")
-		bal, err := cli.QueryBalance("cheqd1lhl9g4rgldadgtz7v6rt50u45uhhj8hhv8d8uf", didtypes.BaseMinimalDenom)
-		Expect(err).To(BeNil())
-		Expect(bal.Amount.Int64()).To(Equal(int64(9000)))
-	})
```

Comment:
This must be properly tested, therefore, please, restore the block + ensure the proper balances are reflected.